app/javascript/components/user/action-creators.js

```diff
@@ -37,10 +37,20 @@ export const fetchAuthenticatedUserData = user => ({
   }
 });
 
+
+export const fetchPrimeroModules = () => ({
+  type: actions.FETCH_RECORD_TYPES,
+  api: {
+    path: "primero_modules/1",
```

Comment:
The hardcoded value of 1 is fine for now as we will replace this by adding record information to the user endpoint.